cmd/test/main_test.go

```diff
@@ -122,6 +124,13 @@ func (lc *LocalConfigMock) GetLocalConfiguration() (*localConfig.ConfigContent,
 	return &localConfig.ConfigContent{CliId: "134kh"}, nil
 }
 
+func pathFromRoot(path string) string {
+	_, filename, _, _ := runtime.Caller(0)
```

Comment:
I like it, I think we should use it in all commands. Nice!

Reply:
thanks :)